Makefile.core.mk

```diff
@@ -152,7 +152,7 @@ test.unit:  ## Run unit tests.
 .PHONY: test.integration
 test.integration: envtest ## Run integration tests located in the tests/integration directory.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" \
-	go test --tags=integration $(VERBOSE_FLAG) ./tests/integration/...
+	go run github.com/onsi/ginkgo/v2/ginkgo --tags=integration $(VERBOSE_FLAG) ./tests/integration/...
```

Comment:
just tried this locally and the output is too much verbose. :(

also, does this run ginkgo from master? This might break us at some point? Should we stick to a released version instead?

Reply:
Yup, too much output due to a bug: https://issues.redhat.com/browse/OSSM-6159

Reply:
As for running ginkgo from master. We use the same approach in the e2e tests. So if this need to be fixed, we should do it in both places (in a follow up PR).

Reply:
Actually, it looks like `go run` honors whatever is in go.mod, so we're actually running v2.15.0.

Reply:
@jwendell I've just pushed another commit that fixes the logs.

Reply:
Much better now. Now I see green dots.

Is is a matter of preference? Because I prefer 1000x the output of `go test`. It's standardized across different projects :)

Reply:
For GInkgo tests, there's only one go test, so if you use `go test`, you don't see anything until the test completes. With ginkgo, you at least see the dots.

Reply:
Oh man, this is so anti pattern :/

Now when we run `make test` we have a mix of go and ginkgo outputs 😩

Reply:
Why anti-pattern? Those are two very different sets of tests. Maybe the anti-pattern is that we run them in a single make target?

Reply:
But that's not really a good argument. Other make targets also have multiple outputs (e.g. docker push + go test output).

Reply:
Sorry, when I said anti-pattern, I was referring to "For GInkgo tests, there's only one go test". I was expecting it to integrate well with the go test subsystem so all ginkgo tests would appear as go tests when run with `go test`...

Reply:
Yeah, I agree, but as you say: "it is what it is" :)

Ginkgo has "specs" not tests. The list of specs gets built from a hierarchy of test "nodes". These specs are then executed (in a single test function that you must write). Unlike functions that are organized into packages, the specs are just one giant sequential list. And they are all executed by your test function.

Reply:
I see. Is there a way to show the progress a little more verbose, instead of just showing small dots?

Reply:
I don't think so. There's only `-v`, but this displays everything that is logged by the test. I don't see any other option that would e.g. only print the specs with nothing else.

Reply:
ok. It is what it is, should we revert to using plain old go tests then? /me runs :D

Reply:
We wouldn't be reverting. We had ginkgo tests from the very start. :)